sa/sa.go

```diff
@@ -1375,11 +1378,38 @@ func (ssa *SQLStorageAuthority) NewAuthorizations2(ctx context.Context, req *sap
 		if err != nil {
 			return nil, err
 		}
-		err = ssa.dbMap.Insert(am)
+
+		// Each authz needs a (?,?...), in the VALUES block
```

Comment:
I need some help understanding this comment.

Reply:
```
		// Each authz needs a (?,?...), in the VALUES block. We need one
		// for each element in the authzFields string with no trailing ','.
```